.github/workflows/test-action.yml

```diff
@@ -261,8 +261,8 @@ jobs:
       statuses: none
     name: Try to install cosign with go
     steps:
-    - uses: actions/checkout@v2
-    - uses: actions/setup-go@v2
+    - uses: actions/checkout@629c2de402a417ea7690ca6ce3f33229e27606a5 # v2
```

Comment:
actually, can you sync to use the same we are using in line 224?

Reply:
Done.

It would be a good idea to have Dependabot monitor and create PRs if using commit hashes. I'll submit a different PR.